Userland/Utilities/uniq.cpp

```diff
@@ -22,13 +22,13 @@ static ErrorOr<void> write_line_content(StringView line, size_t count, bool show
     return {};
 }
 
-static StringView skip(StringView line, unsigned int char_skip_count, unsigned int field_skip_count)
+static StringView skip(StringView line, unsigned char_skip_count, unsigned field_skip_count)
```

Comment:
The code changes in the "Update manpage" commit should be isolated and squashed into the first commit instead.